http_logs/index.json

```diff
@@ -12,38 +12,118 @@
     "properties": {
       "@timestamp": {
         "format": "strict_date_optional_time||epoch_second",
-        "type": "date"
+        "type": "date",
+        "doc_values": {{ timestamp_doc_values | default(true) | tojson }},
+        "index": {{ timestamp_index | default(true) | tojson }}
```

Comment:
I added these and similar statement to be able to test the size of the on disk index after indexing. If we'd like to separate these out into a separate PR I'd be happy with it.

Reply:
I've removed these.